modules/Makefile.circleci

```diff
@@ -30,6 +30,7 @@ circle\:tag:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging $(BUILD)"
 	@$(SELF) DOCKER_TAG=$(BUILD) docker:tag docker:push
+	@$(SELF) docker:tag-git-hash docker:push
```

Comment:
You're right - that would work. I did something similar to that at first, but abandoned that for 2 reasons:
1. The `release-harness` needs to pull down docker images based on git tag, so I needed something like `docker:pull-git-hash` because the `release-harness` should work outside of circle. Since I created a `pull-` target I thought the it nice to have the symmetry with the `docker:tag-git-hash`
2. The `docker:tag-git-hash` target lets me do this in one line: https://github.com/sagansystems/release-harness/pull/12/files#diff-b67911656ef5d18c4ae36cb6741b7965R67  (pull down the docker image corresponding to the git hash, then tag and push it back up again with the release tag)